lib/data.json

```diff
@@ -79,6 +85,7 @@
     "phone" : "(265) 447 2304",
     "email" : "dannyd@wfm.com",
     "avatar" : "https://s3.amazonaws.com/uifaces/faces/twitter/danbenoni/128.jpg",
+    "banner" : "http://www.vajor.com/skin/frontend/default/vajor/images/banner_placeholder_9.png",
```

Comment:
Maybe ist's best to host this somewhere where we have control over it in case it would go down.
Banner looks ugly so maybe we can just get some solid color as example?